corrected manifest.json file

bhuveshkumaroracle opened the pull request:
the links included in manifest.json file are not available.
So there are being removed.

xinyitay commented on oci-library/oci-hol/java-management-oci-services-integration/access-and-utilise-rest-api-via-jms-sdk-postman/access-and-utilise-rest-api-via-jms-sdk-postman.md:
````diff
@@ -28,46 +28,11 @@ In this lab, you will:
-    ```
-
-3. While the text editor is still open, add value for `<unique_ID>` and code to print the response in the end of file.
+4. Add code to print the response at the end of the sample SDK code.
````
Add `the following at the end of the sample SDK code: `

xinyitay commented on oci-library/oci-hol/java-management-oci-services-integration/access-and-utilise-rest-api-via-oci-cli/access-and-utilise-rest-api-via-oci-cli.md:
````diff
@@ -169,23 +169,23 @@ To get **fleet information**, run the following command.
   ```
   <copy>
-  oci jms fleet get --fleet-id "ocid1.jmsfleet.oc1.<unique_ID>"
+  oci jms fleet get --fleet-id "<fleet_OCID>"
````
do we want to include the UI instructions for users to obtain their fleet ocid?

xinyitay commented:
looks good! what do you think about including UI instructions for users to obtain fleet OCID?
otherwise, i think it's good to go :)

bhuveshkumaroracle committed:
corrected manifest.json file

the links included in manifest.json file are not available.
So there are being removed.
